packages/transit-vehicle-overlay/src/components/popups/VehicleTooltip/index.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Tooltip } from "react-leaflet";
+import L from "leaflet";
+
+import { transitVehicleType } from "@opentripplanner/core-utils/src/types";
+import { formatDurationWithSeconds } from "@opentripplanner/core-utils/src/time";
+import { TooltipStyle } from "../styled";
+import { linterIgnoreTheseProps } from "../../../utils";
+
+/** will show a (leaflet) map tooltip on a vehicle, showing route and update recency */
+export default function VehicleTooltip(props) {
+  const { vehicle, isTracked, direction, permanent, offset } = props;
+  linterIgnoreTheseProps(isTracked);
+
+  let rsn = vehicle.routeShortName;
```

Comment:
I don't love the use of this abbreviation `rsn`. I would prefer to see it called `displayName` or `name`.